### PR TITLE
Modify the job to move the logic into a script in common-templates

### PR DESCRIPTION
**Pull request by shwetaap**
Signed-off-by: Shweta Padubidri <spadubid@redhat.com>

Modify the job to move the logic into a script in common-templates.

### CODE REVIEW

File: github/ci/shared-deployments/files/docker-daemon-mirror.conf [hunk 1]
```diff
@@ -1,5 +1,5 @@
 DOCKER_OPTS="${DOCKER_OPTS} --data-root=/docker-graph"
-DOCKER_OPTS="${DOCKER_OPTS} --registry-mirror=http://docker-mirror.kubevirt-prow.svc:5000 --insecure-registry=http://docker-mirror.kubevirt-prow.svc:5000"
```
Comment (fgimenez) [inline github/ci/shared-deployments/files/docker-daemon-mirror.conf:2]: Please remove this change, this is our deployment code for our Prow control plane

**Review by fgimenez — CHANGES_REQUESTED** [review]

**Review by fgimenez — APPROVED** [review]
/lgtm
/approve

Nice job!

**Comment by fgimenez** [comment]
Build passing here https://prow.apps.ovirt.org/view/gcs/kubevirt-prow/logs/periodic-common-templates-nightly-image-builder/1371847801272537088

File: github/ci/shared-deployments/files/docker-daemon-mirror.conf (hunk 1 above)
Comment (shwetaap) [reply github/ci/shared-deployments/files/docker-daemon-mirror.conf]: Done! Thanks!

**Pull request merged**